docs/en/sql-reference/functions/rounding-functions.md

```diff
@@ -26,6 +26,37 @@ Returns the smallest round number that is greater than or equal to `x`. In every
 
 Returns the round number with largest absolute value that has an absolute value less than or equal to `x`‘s. In every other way, it is the same as the ’floor’ function (see above).
 
+**Syntax**
+
+```sql
+trunc(input, precision)
+```
+
+Alias: `truncate`.
+
+**Parameters**
+
+- `input`: A float type [Float](/docs/en/sql-reference/data-types/float.md).
```

Comment:
@peter279k @yariks5s The types for parameters `input` and `precision` were incorrectly documented.

```sql
SELECT trunc(123, 3) as res; -- works and `123` is an integer, not a float

SELECT trunc(123.4, toDecimal32(2, 4)) as res; -- obviously doesn't work but it was documented to work (`a decimal type`)
```

Please check the source code (src/Functions/FunctionsRound.h) and also double-check with fiddle https://fiddle.clickhouse.com/

Reply:
ref. #62720